portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * Fetches content of /data from the server and adds it to the DOM.
+ */
+function getData() {
+  console.log('Fetching data.');
```

Comment:
When code goes to "production", it's better to remove these log statements. They help a lot when developing to help understand the flow of the code, but when it's time to merge to master they can be removed. 

Otherwise your clients will all see these statements in their console.

Reply:
+1

Reply:
Fixed!